options/locale/locale_en-US.ini

```diff
@@ -360,7 +360,7 @@ remember_me = Remember This Device
 forgot_password_title= Forgot Password
 forgot_password = Forgot password?
 sign_up_now = Need an account? Register now.
-sign_up_successful = Account was successfully created.
+sign_up_successful = Account successfully created. Welcome!
```

Comment:
IIRC, the word "create" is used as: "create something" or "something is created"